observatorium/base/observatorium.yaml

```
@@ -0,0 +1,98 @@
+---
+apiVersion: core.observatorium.io/v1alpha1
+kind: Observatorium
+metadata:
+  labels:
+    app.kubernetes.io/name: observatorium-cr
+  name: opf-observatorium
+spec:
+  # Only Loki Components are scaled up
+  loki:
+    image: quay.io/app-sre/loki:1.6.1
+    replicas:
+      distributor: 1
+      ingester: 1
+      querier: 1
+      query_frontend: 1
+      table_manager: 1
+    version: 1.6.1
+    volumeClaimTemplate:
+      spec:
+        accessModes:
+        - ReadWriteOnce
+        resources:
+          requests:
+            storage: 250Mi
+  objectStorageConfig:
+    objectStorageConfig:
+    loki:
+      # FIXME: Most of the following values are not used, yet they are required to be defined
+      # See: https://github.com/observatorium/operator/issues/31
+      accessKeyIdKey: ""
+      bucketsKey: ""
+      endpointKey: endpoint
+      regionKey: ""
+      secretAccessKeyKey: ""
+      secretName: loki-objectstorage
+    thanos:
+      key: thanos.yaml
+      name: thanos-objectstorage
+  # All other Thanos and Observatorium components are scaled down
+  api:
+    rbac:
+      roleBindings:
+      - name: dummy
+        roles:
+        - dummy
+        subjects:
+        - kind: dummy
+          name: dummy
+      roles:
+      - name: dummy
+        permissions:
+        - read
+        - write
+        resources:
+        - logs
+        - metrics
+        tenants:
+        - test
```

Just a quick question. Is this really necessary?

Wouldn't something like be sufficient?

```yaml
rbac: {}
tenants: []
```
I haven't tried it out, it's more of a question. We certainly don't want fixed rbac like this in here, so I'm looking for ways how to avoid it.

---

It is a required parameter in the Observatorium resource

---

Ok, so.. I've cloned your PR and deployed it myself. This works just fine, please use this:

```suggestion
    rbac:
      roleBindings: []
      roles: []
    replicas: 0
    tenants: []
```